internal/deploys/handler_test.go

```diff
@@ -0,0 +1,138 @@
+package deploys
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/broadinstitute/sherlock/internal/builds"
+	"github.com/broadinstitute/sherlock/internal/testutils"
+	"github.com/bxcodec/faker/v3"
+)
+
+func (suite *DeployIntegrationTestSuite) TestCreateDeployHandler() {
+	// setup prepopulate a service instance and some builds
+	var preExistingServiceInstanceReq CreateServiceInstanceRequest
+	err := faker.FakeData(&preExistingServiceInstanceReq)
+	suite.Require().NoError(err)
```

Comment:
conflicted here on consistency vs conciseness, what's your thoughts on 
```suite.Require().NoError(faker.FakeData(&preExistingServiceInstanceReq))```
considering we can't do that for dual output methods.